### ISSUE TITLE
File Reader service

### ISSUE THREAD

**Issue by facetrollex** [facetrollex/fctrlx-file-to-base64#1]
Key points:

- [x] Convert to base64/text/arraybuf methods.
- [x] Response as Promise/Observable
- [x] Move Documentation to wiki, update readme
- [x] Fix Full Example (for some reason lib is not loading on plunkr)

**Comment by mokoro1** [comment]
Hi,

My app.module.ts can't seem to find the module. I'm receiving an error saying:
Cannot find module 'fctrlx-angular-file-reader'.

I've installed the library in the proper procedure that you've shown in the READ.me however there is a high vulnurability issue that was also found:


  
```
$ npm audit
                     === npm audit security report ===
                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance

  High            Arbitrary File Overwrite
  Package         tar
  Patched in      >=4.4.2
  Dependency of   @angular-devkit/build-angular [dev]
  Path            @angular-devkit/build-angular > node-sass > node-gyp > tar
  More info       https://npmjs.com/advisories/803
  found 1 high severity vulnerability in 42620 scanned packages
  1 vulnerability requires manual review. See the full report for details.

```

**Comment by mokoro1** [comment]
> 
> 
> Hi,
> 
> My app.module.ts can't seem to find the module. I'm receiving an error saying:
> Cannot find module 'fctrlx-angular-file-reader'.
> 
> I've installed the library in the proper procedure that you've shown in the READ.me however there is a high vulnurability issue that was also found:
> 
> ```
> $ npm audit
>                      === npm audit security report ===
>                                  Manual Review
>              Some vulnerabilities require your attention to resolve
> 
>           Visit https://go.npm.me/audit-guide for additional guidance
> 
>   High            Arbitrary File Overwrite
>   Package         tar
>   Patched in      >=4.4.2
>   Dependency of   @angular-devkit/build-angular [dev]
>   Path            @angular-devkit/build-angular > node-sass > node-gyp > tar
>   More info       https://npmjs.com/advisories/803
>   found 1 high severity vulnerability in 42620 scanned packages
>   1 vulnerability requires manual review. See the full report for details.
> ```

I managed to find a solution to this, I had to update the package.json for node-gyp.

However there are still some errors I'm gettnig pertaining to the library itself.

![image](https://user-images.githubusercontent.com/44970305/57014133-a11ac300-6c06-11e9-9284-250747b25f0b.PNG)

**Comment by facetrollex** [comment]
@mokoro1 Thanks! I will take a look and will do that in my next "Release".
